src/main/java/frc/robot/Constants.java

```diff
@@ -45,24 +45,24 @@ public static final class DrivetrainConstants {
   }
   public static final class ClimberConstants{
     public static final int rightClawMotor = 5;
-    public static final double rightClawMotorPower = .4;
+    public static final double rightClawMotorPower = .5;
     public static final int leftClawMotor = 6;
-    public static final double leftClawMotorPower = .4;
+    public static final double leftClawMotorPower = .5;
     
     //figure out how to get the limits set up
-    public static final float rightClawLimitIn = -190f;//figure ths out
-    public static final float rightClawLimitOut = 190f; // this will be negated in the subsystem because negative means out
-    public static final float leftClawLimitIn = -190f; //figure this out too
-    public static final float leftClawLimitOut = 190f;
+    public static final float rightClawLimitIn = -400f;//figure ths out
+    public static final float rightClawLimitOut = 400f; // this will be negated in the subsystem because negative means out
+    public static final float leftClawLimitIn = -400f; //figure this out too
+    public static final float leftClawLimitOut = 400f;
   }
   public static final class IntakeConstants {
     public static final int intakeBarMotor = 7;
     public static final double intakeBarMotorPower = 0.3;
     public static final int intakeBeltMotor = 8;
-    public static final double intakeBeltMotorPower = 0.3;
+    public static final double intakeBeltMotorPower = 0.5;
     public static final int trapRollerMotor = 9;
-    public static final double trapRollerMotorPower = 0.3;
-    public static final int switchPort = 0; 
+    public static final double trapRollerMotorPower = -0.3;
+    public static final int switchPort = 9;
```

Comment:
it doesn't look like this switchPort is used anywhere. Can we delete this line?

Reply:
I think so? I will take it out and hopefully it doesn't mess anything up -- I think it was supposed to be the port for the limit switch but that's actually somewhere else.